opendm/config.py

```diff
@@ -147,6 +147,40 @@ def config():
                         help=('The maximum number of processes to use in dense '
                               'reconstruction. Default: %(default)s'))
 
+    parser.add_argument('--opensfm-depthmap-resolution',
+                        metavar='<positive float>',
+                        type=float,
+                        default=640,
+                        help=('Resolution of the depthmaps. Higher values take longer to compute '
+                              'but produce denser point clouds. '
+                              'Default: %(default)s'))
+
+    parser.add_argument('--opensfm-depthmap-min-consistent-views',
+                      metavar='<integer: 2 <= x <= 9>',
+                      type=int,
+                      default=3,
+                      help=('Minimum number of views that should reconstruct a point for it to be valid. Use lower values '
+                            'if your images have few overlap. Lower values result in denser point clouds '
```

Comment:
I think the grammatically correct word here is less, this is the only nitpicky comment i can make because the rest of this looks great!

Reply:
We care about grammar, so I'll fix it.